controllers/templates/devdb.tmpl

```diff
@@ -51,6 +55,15 @@ spec:
               value: pass
             - name: POSTGRES_USER
               value: root
+          {{- else if eq .Driver "sqlserver" }}
+          image: mcr.microsoft.com/mssql/server:2022-latest
+          env:
+            - name: ACCEPT_EULA
+              value: "Y"
```

Comment:
We need some way where the user accepts the Microsoft EULA, we cannot accept it for them